src/scene/model/mod.rs

```diff
@@ -237,10 +272,17 @@ impl Model {
     }
 
     pub(crate) fn load(
+        mesh: &gltf::Mesh,
+        primitive_index: usize,
         primitive: gltf::Primitive,
         transform: &Matrix4<f32>,
         data: &mut GltfData,
     ) -> Self {
+        #[cfg(not(feature="names"))]
+        {
+            let _ = mesh;
+        }
```

Comment:
Instead, rename `mesh` argument to `_mesh` this will disable warnings if "names" feature isn't used.

Reply:
The underscore silences other errors though. Doing an unused assignment in a condition silences the unused variable warning only in the case where `names` is off, while keeping every other warning active (including the unused variable warning when `names` is on and the variable should be used.